examples/index_robust04.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+PISA_PREFIX=~/src/pisa
+PISA_BIN=$PISA_PREFIX/build/bin
+THREADS=20
+
+COLLECTION=(/research/remote/collections/TREC/disk4/fr.gz
+	    /research/remote/collections/TREC/disk4/ft.gz
+	    /research/remote/collections/TREC/disk5/fbis.gz
+	    /research/remote/collections/TREC/disk5/latimes.gz)
+
+ROB_PREFIX=./robust04
+
+# Generate robust04 robust04.terms robust04.urls robust04.documents
+zcat ${COLLECTION[@]} | $PISA_BIN/parse_collection -j $THREADS -f trectext --content-parser html --stemmer krovetz -o $ROB_PREFIX
```

Comment:
Did you test the krovetz stemmer? How is it working for you?

Reply:
It worked pretty well for me as far as I can tell according to the end-to-end output.

I got crash when stemming some tokens I manually parsed but I didn't dig into that so not sure what happened exactly.

Reply:
```

Reply:
```
$ wc -l plaintext 
1000 plaintext
$ cat plaintext | parse_collection --batch-size 500 --output ./discard --stemmer krovetz ; rm -f discard*
[2019-08-23 12:18:23.720] [info] Number of threads: 56
Segmentation fault (core dumped)
```

```
(gdb) bt                                                                                                                                                      
#0  0x00007f797c23a244 in std::__1::basic_istream<char, std::__1::char_traits<char> >::sentry::sentry(std::__1::basic_istream<char, std::__1::char_traits<char
> >&, bool) () from /home/sl8/S3676608/.local/lib/libc++.so.1
#1  0x000000000047bd70 in std::__1::operator>><char, std::__1::char_traits<char>, std::__1::allocator<char> > (__is=..., __str=...)
    at /home/sl8/S3676608/.local/bin/../include/c++/v1/istream:1336                                                                                           
#2  0x000000000047488b in pisa::parse_plaintext_content(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&&, std::__1::fun
ction<void (std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&&)>) (content=<optimized out>, process=...)
    at /research/remote/petabyte/users/binsheng/pisa/include/pisa/forward_index_builder.hpp:63
```

The crash turns out to be caused by clang's libc++. Seems to be an upstream bug.